src/components/PageHeader.tsx

```diff
@@ -59,14 +60,24 @@ export const PageHeader = ({ title }: Props) => {
           </a>
           <div className={classes.title}>{title}</div>
         </div>
-        <a
-          className={classes.contact}
-          href={process.env.REACT_APP_FIDER_LINK}
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Contact
-        </a>
+        <div>
+          <a
+            className={classes.contact}
```

Comment:
We haven't decided that we're going to have a link to the fire weather calculator, and it's not part of the acceptance criteria of this story.

Reply:
@Sybrand the link is for temporary purpose as it would be removed as part of next coming PRs. The link was only introduced to showcase the navigation and login functionality.

Reply:
We can demonstrate this by linking directly to a url know to us - please remove it from this PR as it ads unwanted functionality.

Reply:
ok will remove it.